Day3/Java/HammingDistancePrabhat.java

```diff
@@ -0,0 +1,39 @@
+/* @date 03/10/2020
+ * @author Prabhat Malhotra
+ */
+
+import org.junit.jupiter.api.Test;
+
+/*
+ * Solution
+ */
+public class HammingDistancePrabhat {
+
+    public int calculateHammingDistance(String firstString, String secondString){
+        int totalChange = 0;
+        byte[] aByteArray = firstString.getBytes();
+        byte[] bByteArray = secondString.getBytes();
+
+        for(int i = 0; i < aByteArray.length; i++){
```

Comment:
You need to first check if the strings are of the same length. Hamming distance is undefined for strings of different lengths.